src/main/game/Line.java

```diff
@@ -5,17 +5,21 @@
 import static java.util.Arrays.asList;
 
 public class Line {
-    private List<String> spaces;
+    private List<Mark> spaces;
 
-    public Line(String spaceOne, String spaceTwo, String spaceThree) {
+    public Line(Mark spaceOne, Mark spaceTwo, Mark spaceThree) {
```

Comment:
if change the constructor to take a list instead of three (four) individual items you can grow to AAAAANY Line you want!